File: Flight Management System/flight/src/main/java/com/dxc/flightmodule/flight/entity/Flight.java
package com.dxc.flightmodule.flight.entity;

import java.math.BigInteger;
import java.util.Objects;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "Flight_Details")
public class Flight {

	@Id
	@GeneratedValue
	private BigInteger flightNumber;

	private String flightModel;

	private String carrierName;

	private Integer seatCapacity;

	public Flight() {
	}

	public Flight(BigInteger flightNumber, String flightModel, String carrierName, Integer seatCapacity) {
		this.flightNumber = flightNumber;
		this.flightModel = flightModel;
		this.carrierName = carrierName;
		this.seatCapacity = seatCapacity;
	}

	public BigInteger getFlightNumber() {
		return flightNumber;
	}

	public void setFlightNumber(BigInteger flightNumber) {
		this.flightNumber = flightNumber;
	}

	public String getFlightModel() {
		return flightModel;
	}

	public void setFlightModel(String flightModel) {
		this.flightModel = flightModel;
	}

	public String getCarrierName() {
		return carrierName;
	}

	public void setCarrierName(String carrierName) {
		this.carrierName = carrierName;
	}

	public Integer getSeatCapacity() {
		return seatCapacity;
	}

	public void setSeatCapacity(Integer seatCapacity) {
		this.seatCapacity = seatCapacity;
	}

	@Override
	public int hashCode() {
		return Objects.hash(flightNumber);
	}

	@Override
	public boolean equals(Object obj) {
		if (this == obj) {
			return true;
		}
		if (obj == null || getClass() != obj.getClass()) {
			return false;
		}
		Flight that = (Flight) obj;
		return this.flightNumber.equals(that.flightNumber);
	}

}

File: flight-mgt-system/src/app/model/flight.ts
export class Flight{
    flightNumber:number;
    flightModel:string;
    carrierName:string;
    seatCapacity:number;

    constructor(flightNumber:number,flightModel:string,carrierName:string,seatCapacity:number){
        this.flightNumber=flightNumber;
        this.flightModel=flightModel;
        this.carrierName=carrierName;
        this.seatCapacity=seatCapacity;
    }

    getFlightNumber():number{
        return this.flightNumber;
    }

    getFlightModel():string{
        return this.flightModel;
    }

    getCarrierName():string{
        return this.carrierName;
    }

    getSeatCapacity():number{
        return this.seatCapacity;
    }

}